grails-app/services/vincool/cloudinary/CloudinaryUploadResult.groovy

```diff
@@ -0,0 +1,37 @@
+package vincool.cloudinary
+
+
+class CloudinaryUploadResult {
+
+    String publicId
+    long version
+    String signature
+    int width
+    int height
+    String format
+    String resourceType
+    Date createdAt
+    long bytes
+    String type
+    String url
+    String secureUrl
+
+
+    @Override
+    public String toString() {
```

Comment:
If this is a required way of to string it's ok. If this is just debugging maybe it should be better to use @ToString http://mrhaki.blogspot.mx/2011/04/groovy-goodness-easy-tostring-creation.html